face_api_dataset/dataset.py

```diff
@@ -845,4 +846,16 @@ def _read_landmarks_3d(cls, info: dict, mdt: Modality) -> Dict[Id, Landmark_3D]:
             lmk_name = landmark["id"]            
             landmarks[int(lmk_name)] = tuple(landmark["camera_space_pos"])
 
-        return landmarks
\ No newline at end of file
+        return landmarks
+    
+    @classmethod
```

Comment:
Please, add docstings to the new methods and regenerate the docs (make html will do the trick).